types/node/test/test.ts

```diff
@@ -768,6 +778,24 @@ test("mocks a setter", (t) => {
     }
 });
 
+test("mocks a module", (t) => {
+    const mock = t.mock.module("node:readline", {
```

Comment:
```suggestion
    // $ExpectType MockModuleContext
    const mock = t.mock.module("node:readline", {
```
Might be worth an explicit type check here.